lib/puppet/provider/service/redhat.rb

```diff
@@ -39,6 +41,7 @@ def enabled?
   # Don't support them specifying runlevels; always use the runlevels
   # in the init scripts.
   def enable
+      chkconfig("--add", @resource[:name])
```

Comment:
@joshcooper My note about the --add being harmless on unaffected OS versions refers to the fact that the bug exists on CentOS/RHEL 5 but not v6, where the acceptance test is also being run. I did give some thought to running it on v7 as well, but I haven't found a good service to use for testing that uses chkconfig and not systemd on v7. My search hasn't been 100% exhaustive, and I can look further into this on Monday.